test/spectests/spectests.cpp

```diff
@@ -282,6 +310,55 @@ class test_runner
         return fizzy::execute(instance, *func_idx, std::move(args));
     }
 
+    std::optional<imports> create_imports(const fizzy::Module& module)
+    {
+        std::vector<fizzy::ImportedFunction> imported_functions;
+        std::vector<fizzy::ImportedGlobal> imported_globals;
+        for (auto const& import : module.importsec)
+        {
+            const auto it_registered = registered_names.find(import.module);
+            if (it_registered == registered_names.end())
+            {
+                fail("Module \"" + import.module + "\" not registered.");
+                return std::nullopt;
+            }
+
+            const auto module_name = it_registered->second;
+            const auto it_instance = instances.find(module_name);
+            if (it_instance == instances.end())
+            {
+                fail("Module not instantiated.");
+                return std::nullopt;
+            }
+
+            auto& instance = it_instance->second;
+
+            if (import.kind == fizzy::ExternalKind::Function)
+            {
+                const auto func_idx = fizzy::find_exported_function(instance.module, import.name);
```

Comment:
Should use `find_export_function(instance, name)` and also simplify the code below.

Reply:
Yeah this is done already in https://github.com/wasmx/fizzy/pull/182

I will split `spectest` module into a separate PR, leave this one with `register` command only and #182 with all the imports support.